store/src/java/com/zimbra/cs/account/ZimbraAuthToken.java

```diff
@@ -33,8 +34,12 @@
 import org.apache.commons.httpclient.HttpMethod;
 import org.apache.commons.httpclient.HttpState;
 import org.apache.commons.httpclient.cookie.CookiePolicy;
+import org.apache.commons.rng.UniformRandomProvider;
```

Comment:
Can you please confirm there is not any vulnerability reported against version of libraries we are using here?